src/data/openthaigpt_pretraining_data/mc4/preprocess.py

```diff
@@ -0,0 +1,251 @@
+# flake8: noqa
+from pattern import (
+    toolarge_re,
```

Comment:
If possible, please write these pattern in uppercase like TOOLARGE_RE

Reply:
OK, krub